### PR TITLE
Use `<button>` instead of `<div>` in `BetaFish` demo

### PR DESCRIPTION
**Pull request by alrra**
Fix MicrosoftEdge/Demos#185.

### CODE REVIEW

File: betafish/Script/Settings.js [hunk 1]
```diff
@@ -3,15 +3,23 @@
...
+    var toggleSwimmingButton = document.getElementById("ButtonToggleSwimming");
+
     if (fishSwimming === true) {
         fishSwimming = false;
         document.getElementById("Fish").className = "FishSwimming Paused";
-        document.getElementById("ButtonToggleSwimming").style.backgroundImage = "url('Images/ButtonPlay.png')";
+        toggleSwimmingButton.setAttribute("aria-label", "Play demo");
+        toggleSwimmingButton.title = "Play (P)";
+        toggleSwimmingButton.style.backgroundImage = "url('Images/ButtonPlay.png')";
+
```
Comment (sarvaje) [inline betafish/Script/Settings.js]: remove this empty line please

**Review by sarvaje — CHANGES_REQUESTED** [review]

File: betafish/Script/Settings.js (hunk 1 above)
Comment (alrra) [reply betafish/Script/Settings.js]: @sarvaje Done!

**Review by sarvaje — APPROVED** [review]

**Comment by sarvaje** [comment]
LGTM

**Comment by sarvaje** [comment]
@melanierichards or @dstorey can you validate/merge this PR?

**Comment by melanierichards** [comment]
Shippin it!

**Pull request merged**